Sources/AnimalKingdomAPI/AnimalKingdomAPI/Sources/Schema/CustomScalars/CustomDate.scalar.swift

```diff
@@ -0,0 +1,10 @@
+// @generated
```

Comment:
The old versions of these scalar files need to be deleted. But this actually surfaces a bigger issue here.

If a user had this new config `false` and then sets it to true, previously generated custom scalars (and any other  files that are marked to not be overwritten) will not be handled properly. We need to preserve the changes to the code the user may have made. So we actually need to detect the old file name and _rename_ the existing file, rather than just writing a new file.

Reply:
OK, this is now done too. Logic in https://github.com/apollographql/apollo-ios-dev/pull/580/commits/6a22d3767ea74d77b426d3d55366bd3fc68f6444 and tests in https://github.com/apollographql/apollo-ios-dev/pull/580/commits/5743c59c1c77de0200c08776cdb045ab0b2ff5ee.